tensorflow/core/platform/s3/s3_file_system.cc

```diff
@@ -82,6 +87,43 @@ Aws::Client::ClientConfiguration& GetDefaultClientConfig() {
   return cfg;
 };
 
+Aws::Utils::Logging::LogLevel ParseLogLevelFromEnv() {
+  Aws::Utils::Logging::LogLevel logLevel = Aws::Utils::Logging::LogLevel::Info;
+
+  const char* tf_env_var_val = getenv("TF_CPP_MIN_LOG_LEVEL");
```

Comment:
Can you look into sharing code with `tensorflow::internal::MinLogLevelFromEnv`?

Reply:
Thanks @drpngx. Fixed.